auth/opa/rpcauth/rpcauth_test.go

```diff
@@ -138,6 +139,8 @@ func TestAuthzHook(t *testing.T) {
 		},
 	})
 	testutil.FatalOnErr("json.Marshal extensions", err, t)
+	emptyMessage := emptypb.Empty{}
+	marshaledEmptyMessage, _ := protojson.MarshalOptions{UseProtoNames: true}.Marshal(&emptyMessage)
```

Comment:
The tests might be a bit clearer without this indirection. `[]byte("{}")` should be a valid value.

Reply:
ah I didn't know. thanks!